airsonic-main/src/main/resources/org/airsonic/player/theme/2010.properties

```diff
@@ -12,33 +12,33 @@ detailColor = AAAAAA
 linkColor = FFFFFF
 
 # Set the images for theme.
-addImage = /icons/2010/add.png
-clearRatingImage = /icons/2010/clear_rating.png
-currentImage = /icons/2010/now_playing.png
-donateImage = /icons/2010/donate.png
-downImage = /icons/2010/down.png
-downloadImage = /icons/2010/download.png
-helpImage = /icons/2010/help.png
-helpPopupImage = /icons/2010/help_small.png
-homeImage = /icons/2010/home.png
-logImage = /icons/2010/log.png
-logoImage = /icons/2010/logo.png
-moreImage = /icons/2010/more.png
-nowPlayingImage = /icons/2010/playing.png
-paypalImage = /icons/2010/paypal.gif
-playImage = /icons/2010/play.png
-podcastImage = /icons/2010/podcast_small.png
-podcastLargeImage = /icons/2010/podcast.png
-randomImage = /icons/2010/random.png
-ratingOnImage = /icons/2010/rating_on.png
-ratingOffImage = /icons/2010/rating_off.png
-ratingHalfImage = /icons/2010/rating_half.png
-removeImage = /icons/2010/remove.png
-searchImage = /icons/2010/search.png
-settingsImage = /icons/2010/settings.png
-statusImage = /icons/2010/status.png
-upImage = /icons/2010/up.png
-uploadImage = /icons/2010/upload.png
-volumeImage = /icons/2010/now_playing.png
-wapImage = /icons/2010/phone.png
+addImage = icons/2010/add.png
+clearRatingImage = icons/2010/clear_rating.png
```

Comment:
I think this entire commit can be removed once you rebase to remove the original undesired change.